runtime/pom.xml

```diff
@@ -140,10 +154,34 @@
                         </execution>
                     </executions>
                 </plugin>
+                <plugin>
+                    <groupId>org.apache.maven.plugins</groupId>
+                    <artifactId>maven-assembly-plugin</artifactId>
```

Comment:
assembly plugin is an option, we can also be simpler by using shade plugin.

Reply:
switched to shade plugin.